RiotFrontend/Providers/MatchProvider.cs

```diff
@@ -55,6 +56,27 @@ public Match GetMatch(string matchId)
             return dtoConverter.GetMatchContract(matchInfo, FormatType.Detailed);
         }
 
+        public Match GetMatch(string matchId, string summonerName)
+        {
+            Validation.ValidateNotNullOrWhitespace(matchId, nameof(matchId));
+            Validation.ValidateNotNullOrWhitespace(summonerName, nameof(summonerName));
+
+            var rowKeyFilter = TableQuery.GenerateFilterCondition(RowKeyColumn, QueryComparisons.Equal, matchId);
+            var partitionKeyFilter = TableQuery.GenerateFilterCondition(PartitionKeyColumn, QueryComparisons.Equal, summonerName.ToLowerInvariant());
+            var finalFilter = TableQuery.CombineFilters(rowKeyFilter, TableOperators.And, partitionKeyFilter);
```

Comment:
You should be able to omit the final filter; you can pass rowKeyFilter and partitionKeyFitler to GetRows()